Packs/FeedProofPoint/Integrations/FeedProofPoint/FeedProofPoint.py

```diff
@@ -0,0 +1,166 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+""" IMPORTS """
+import urllib3
+import csv
+from typing import Generator, Tuple, Optional, List
+
+# disable insecure warnings
+urllib3.disable_warnings()
+
+SOURCE_NAME = "Proofpoint Feed"
+
+
+class Client(BaseClient):
+    def __init__(self, base_url, auth_code, **kwargs):
+        base_url = url_concat(base_url, auth_code, "reputation")
+        super().__init__(base_url, **kwargs)
+
+    DOMAIN_TYPE = "domain"
+    IP_TYPE = "ip"
+    ALL_TYPE = "all"
+    TYPES = (DOMAIN_TYPE, IP_TYPE, ALL_TYPE)
+    indicator_types_to_endpoint = {
+        IP_TYPE: ["detailed-iprepdata.txt"],
+        DOMAIN_TYPE: ["detailed-domainrepdata.txt"],
+    }
+
+    def _build_iterator(
+        self, indicator_type: str = "all"
+    ) -> Generator[dict, None, None]:
+        endpoints = self.indicator_types_to_endpoint.get(
+            indicator_type, self.indicator_types_to_endpoint.values()
+        )
+        for endpoint in endpoints:
+            resp = self._http_request(
+                "GET", endpoint, resp_type="text", timeout=(30, 60)
+            )
+            resp = resp.splitlines()
+            csv_repr = csv.reader(resp)
+            headers: list = next(csv_repr, None)
+            headers = [header.replace(" ", "").replace("(|)", "") for header in headers]
+            for line in csv_repr:
+                yield {headers[i]: line[i] for i in range(len(headers))}
+
+    def _build_iterator_domain(self) -> Generator[dict, None, None]:
+        """Gives the user an
+
+        Returns:
+
+        """
+        return self._build_iterator(self.DOMAIN_TYPE)
+
+    def _build_iterator_ip(self) -> Generator[dict, None, None]:
+        return self._build_iterator("ip")
+
+    def get_indicators_domain(self) -> List[dict]:
+        return [
+            {
+                "value": item["domain"],
```

Comment:
It appears that part of the data returned is a score - translate the scores to dbotscores and then return it in a field called `score`.